packages/facility-server/app/routes/apiv1/patient/patientprogramRegistration/patientProgramRegistrationConditions.js

```diff
@@ -0,0 +1,110 @@
+import express from 'express';
```

Comment:
All of the endpoints in this file are unchanged

Reply:
Thanks for breaking these out into a separate file :pray: